yearn/apy/v2.py

```diff
@@ -68,9 +67,28 @@ def simple(vault, samples: ApySamples) -> Apy:
     apys = [week_ago_apy, month_ago_apy, inception_apy]
     net_apy = next((value for value in apys if value != 0), 0)
 
-    # performance fee is doubled since 1x strategists + 1x treasury
-    performance = (contract.performanceFee() * 2) if hasattr(contract, "performanceFee") else 0
+    # for performance fee, half comes from strategy (strategist share) and half from the vault (treasury share)
+    strategy_fees = []
+    for x in range(20): # we can have up to 20 strategies
+        try:
+            strategy_to_check = contract.withdrawalQueue(x)
+            print("Strategy to check", strategy_to_check)
+        except AttributeError:
+            break
+        if strategy_to_check == "0x0000000000000000000000000000000000000000":
+            print("No more strategies")
+            break
+        else:
+            print("This is our strategy", strategy_to_check)
+            debt_ratio = contract.strategies(strategy_to_check)[2] / 10000
+            performance_fee = contract.strategies(strategy_to_check)[0]
```

Comment:
Use named struct fields for better readability.
```suggestion
            debt_ratio = contract.strategies(strategy_to_check)['debtRatio'] / 10000
            performance_fee = contract.strategies(strategy_to_check)['performanceFee']
```